File: loops/fizzbuzz.c
 //johann hines, template
#include <stdio.h>

int(x) =1;
int main(void){
while (x >0 && x < 51){
  if (x % 5 == 0 && x % 3 == 0){
    printf("fizzbuzz\n");
  }else if (x % 5 == 0){
    printf("buzz\n");
  }else if (x % 3 == 0){
    printf("fizz\n");
  }else{
    printf("%d\n", x);
  }
  x +=1;
}
    return 0;
}

  

File: loops/fizzbuzz.py
#Johann Hines, Fizzbuzz.py

x =1 

while x >= 1 and x <51:
  if x % 15 == 0:
    print("fizzbuzz.\n")
  elif x % 5 == 0:
    print("buzz.\n")
  elif x % 3 == 0:
    print("fizz\n")
  else:
    print(x)
  x += 1 
